src/components/ListItem.jsx

```diff
@@ -1,5 +1,113 @@
 import './ListItem.css';
+import { updateItem } from '../api/firebase';
+import { useState, useEffect } from 'react';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ name, itemId, dateLastPurchased }) {
+	const [check, setCheck] = useState(false);
+
+	/**
+	 * When List view is opened or refreshed,
+	 * this verifies how long since the item was
+	 * purchased and checks the box if the item
+	 * was purchased fewer than 24 hours ago.
+	 */
+	useEffect(() => {
+		const currentDate = new Date();
+		let purchasedDate = dateLastPurchased
+			? dateLastPurchased.toDate()
+			: dateLastPurchased;
+		const timeElapsed = Math.abs(currentDate - purchasedDate);
+		const hoursElapsed = timeElapsed / (1000 * 60 * 60);
+		hoursElapsed < 24 ? setCheck(true) : setCheck(false);
+	}, []);
+
+	const handleCheck = async (e) => {
```

Comment:
I saw the to-do item in the PR description. I didn't read anything in the AC that talks about what happens when the user unchecks a box. I think you should either 1) disable the checkbox if it's rendered as checked with 
`disabled="true"` so you don't have this edge case, or 2) when the user unchecks the checkbox, copy the timestamp to a property that doesn't affect the UI so that you can retrieve it if you need to.

Truth is, this all feels like a checkbox doing a button's job :-P

Reply:
@mtfoley Copying the timestamp is definitely what we were having a lot of issue with. We tried doing that in ‘updateItem’ in ‘firebase.js’ but were unsuccessful. There’s probably a much simpler way than what we attempted there 😅